src/app/code/community/Omise/Gateway/Model/PaymentMethod.php

```diff
@@ -28,24 +25,21 @@ public function authorize(Varien_Object $payment, $amount)
     {
         Mage::log('Start authorize with OmiseCharge API!');
         
+        $order = $payment->getOrder();
         $omise_token = $payment->getData('additional_information');
-        $charge = Mage::getModel('omise_gateway/omiseCharge')->createOmiseCharge(array(
+        $charge = Mage::getModel('omise_gateway/omisecharge')->createOmiseCharge(array(
```

Comment:
@duzydook please change this line `Mage::getModel('omise_gateway/omisecharge')` to `Mage::getModel('omise_gateway/omiseCharge')` krub (case-sensitive class name reason)